apps/admin_app/lib/admin_app_web/controllers/product_controller.ex

```diff
@@ -75,7 +75,7 @@ defmodule AdminAppWeb.ProductController do
   def update(conn, %{"product" => params}) do
     with %ProductSchema{} = product <- ProductModel.get(params["id"]),
          {:ok, _product} <- ProductModel.update(product, params) do
-      redirect_with_updated_conn(conn, params)
+      redirect_with_updated_conn(conn, @rummage_default)
```

Comment:
rummage_default is one solution, but I want the functionality if the user has done edit, with rummage opts, sort as query params, then those needs to be preserved, which in turn will do the redirection with same filters as were before not the defaults.